### DISCUSSION TITLE
Select next completion item

### DISCUSSION THREAD

**Discussion by dzfrias** [tridactyl/tridactyl#4684] [Q&A] [answered]
When the excmd popup appears (or in `tabopen` etc.), `<Tab>` is used to sift through the completion items. Is there a way to rebind this?

After scrolling through the documentation for a bit, I'm not sure where I'd find such a rebinding.

**Comment by bovine3dom** [comment, marked as answer]
`:bind --mode=ex [key sequence] ex.next_completion`

**Comment by dzfrias** [comment]
Works great, thanks!
